### PR TITLE
Ev: Add map-reduce style interface.

### PR DESCRIPTION
**Pull request by jaheba**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup

### CODE REVIEW

File: src/gluonts/torch/__init__.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "LagTSTEstimator",
 ]
 import torch
+<<<<<<< Updated upstream
```
Comment (lostella) [inline src/gluonts/torch/__init__.py]: slipped in
